New native vs cross platform

Daria-Voronina opened the pull request:
Please check the new article about native and cross-platform mobile development.

koshachy commented on docs/topics/native-and-cross-platform.md:
```diff
@@ -0,0 +1,130 @@
+
+In fact, it’s the opposite. Cross-platform solutions help you more effectively manage resources. Your development teams can learn how to write and reuse shared code. Android and iOS developers can achieve high efficiency and transparency by interacting and sharing knowledge.
+
+## 6 key aspects to help you choose between cross-platform app development and the native approach
```
It's better to leave text instead of digit here: 
* it will be easier to read
* it looks like a part of a list before but it actually isn't

koshachy commented on docs/topics/native-and-cross-platform.md:
```diff
@@ -0,0 +1,130 @@
+
+Poor performance was long considered to be one of the main disadvantages of multiplatform applications. However, the performance and quality of your product largely depend on the tools you use to build the app. The latest cross-platform frameworks provide all the tools necessary to develop apps with a native-like user experience.
+
+By using different compiler backends, [Kotlin](https://kotlinlang.org/docs/multiplatform.html#code-sharing-between-platforms) is compiled to platform formats – JVM bytecode for Android and native binaries for iOS. As a result, the performance of your shared code is the same as if you write them natively.
```
```suggestion
By using different compiler backends, [Kotlin](multiplatform.md#code-sharing-between-platforms) is compiled to platform formats – JVM bytecode for Android and native binaries for iOS. As a result, the performance of your shared code is the same as if you write them natively.
```

koshachy commented on docs/topics/native-and-cross-platform.md:
```diff
@@ -0,0 +1,130 @@
+* You want to have a single codebase for the app logic while keeping full control over UI elements. Not all cross-platform frameworks allow you to do this, but some, like Kotlin Multiplatform Mobile, provide this capability.
+* You’re eager to embrace a modern technology that continues to evolve.
+
+> Share the logic of your iOS and Android apps. See [Kotlin Multiplatform Mobile](https://kotlinlang.org/docs/multiplatform-mobile-getting-started.html) in action.
```
```suggestion
> Share the logic of your iOS and Android apps. See [Kotlin Multiplatform Mobile](multiplatform-mobile-getting-started.md) in action.
```

koshachy reviewed (approved):
Overall is quite good!
Some minor comments, please take a look, then you could merge it.

Daria-Voronina commented:
I've fixed issues and added a link to the first article (cross-platform mobile development) in the text. As far as I understand, now I can squash and merge it?
